.travis.yml

```diff
@@ -207,6 +207,9 @@ before_cache:
    esac
    docker save -o $HOME/docker_cache/images.tar $(docker images -a -q)
 
+after_success:
+  - scripts/travis/upload_coverage.sh
```

Comment:
I think it would be better to do
```bash
scripts/travis/upload_coverage.sh||true
```
so that the build won't fail in case we have a code coverage report uploading error.
( i.e. to make sure it doesn't break the build )